apps/codeforafrica/content/settings/footer.md

```diff
@@ -1,86 +1,72 @@
+  This site is a project of Code for Africa, the continent's largest network of
+  civic technology and data journalism labs. All content is released under a
+  Creative Commons 4 Attribution Licence. Reuse it to help empower your own
+  community.
 stay-in-touch:
   title: STAY IN TOUCH
   links:
-    - link:
-        content: Twitter
-        href: https://twitter.com/Code4Africa
-        icon:
-          src: /images/type-twitter-size-32-color-white.svg
-    - link:
-        content: Slack
-        href: https://code4africa.slack.com/
-        icon:
-          src: /images/type-slack-size-64-color-currentcolor.svg
-    - link:
-        content: Linkedin
-        href: https://www.linkedin.com/company/code-for-africa/mycompany/
-        icon:
-          src: /images/type-linkedin-size-32-color-white.svg
-    - link:
-        content: Facebook
-        href: https://www.facebook.com/CodeForAfrica/
-        icon:
-          src: /images/type-facebook-size-32-color-white.svg
-    - link:
-        content: Instagram
-        href: https://www.instagram.com/code4africa__/
-        icon:
-          src: /images/type-instagram-size-32-color-white.svg
-    - link:
-        content: Github
-        href: https://github.com/CodeForAfrica
-        icon:
-          src: /images/type-github-size-32-color-white.svg
+    - label: Twitter
+      href: https://twitter.com/Code4Africa
+      icon:
+        src: /images/type-twitter-size-32-color-white.svg
+    - label: Slack
+      href: https://cfa.slack.com/
+      icon:
+        src: /images/type-slack-size-32-color-white.svg
+    - label: Linkedin
+      href: https://www.linkedin.com/company/code-for-africa/mycompany/
+      icon:
+        src: /images/type-linkedin-size-32-color-white.svg
+    - label: Facebook
+      href: https://www.facebook.com/CodeForAfrica/
+      icon:
+        src: /images/type-facebook-size-32-color-white.svg
+    - label: Instagram
+      href: https://www.instagram.com/code4africa__/
+      icon:
+        src: /images/type-instagram-size-32-color-white.svg
+    - label: Github
+      href: https://github.com/CodeForAfrica
+      icon:
+        src: /images/type-github-size-32-color-white.svg
 main-navigation:
-  - link:
-      content: Our work
-      href: /projects
-  - link:
-      content: About
-      href: /about
-  - link:
-      content: Stories
-      href: stories
-  - link:
-      content: Opportunities
-      href: /opportunities
-  - link:
-      href: /contact
-      content: Contact
+  - label: Our work
+    href: /projects
+  - label: About
+    href: /about
+  - label: Stories
+    href: /stories
+  - label: Opportunities
+    href: /opportunities
+  - label: Contact
+    href: /contact
 secondary-navigation:
-  - link:
-      content: Imprint
-      href: /imprint
-  - link:
-      content: Privacy policy
-      href: /privacy
+  - label: Imprint
+    href: /imprint
+  - label: Privacy policy
+    href: /privacy
 newsletter-subscription:
-  title: Sign Up
```

Comment:
`NewsletterSubscription` should have a title

![Screenshot from 2022-08-04 15-11-45](https://user-images.githubusercontent.com/1779590/182847535-2e2d1168-3e91-4e17-b47c-78097b7bddb8.png)